definitions/scenarios/restore.rb

```diff
@@ -45,17 +41,16 @@ def compose
       if backup.sql_dump_files_exist? && feature(:instance).postgresql_local?
         add_step(Procedures::Service::Stop.new(:only => ['postgresql']))
       end
-      restore_mongo_dump(backup)
 
       if feature(:instance).postgresql_local? && !backup.online_backup?
         add_step_with_context(Procedures::Restore::ReindexDatabases)
       end
 
-      add_steps_with_context(Procedures::Pulp::Migrate,
+      add_steps_with_context(
         Procedures::Pulpcore::Migrate,
-        Procedures::Restore::CandlepinResetMigrations)
+        Procedures::Restore::CandlepinResetMigrations
+      )
 
-      add_steps_with_context(Procedures::Restore::RegenerateQueues) if backup.online_backup?
```

Comment:
why is this removed? it doesn't seem pulp2 related?

Reply:
Solid question, according to https://github.com/theforeman/foreman_maintain/pull/707/files#diff-f9d25557678350ba1c6282e05ea917d8eb58185b49203038a43aba238c149b56L6-L8 this was confined to Pulp 2 only, so it has not been used for quite a few releases now.

Reply:
amazing(ly confusing)